cypress/e2e/deputies/deputies_list.cy.js

```diff
@@ -22,9 +22,25 @@ describe("PA Deputies list", () => {
         })
     })
 
-    it("can be sorted by Non-compliance, Deputy & Assurance", () => {
+    it("can be sorted by Deputy, Active clients, Non-compliance & Assurance", () => {
         cy.url().should("not.contain", "order-by").and("not.contain", "sort")
 
+        cy.get("th:nth-child(2) button").click()
```

Comment:
Would it be better to grab the button by a string (e.g. Contains "deputy") or id?